docs/data-sources/service_level_objective.md

```diff
@@ -0,0 +1,30 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_service_level_objective Data Source - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Use this data source to retrieve information about an existing SLO for use in other resources.
+---
+
+# datadog_service_level_objective (Data Source)
+
+Use this data source to retrieve information about an existing SLO for use in other resources.
+
+
```

Comment:
To get an HCL config generated here, can you add an example to the examples/data-source folder?